MdeModulePkg/Bus/Pci/NvmExpressDxe/UnitTest/MediaSanitizeUnitTest.c

```diff
@@ -0,0 +1,1208 @@
+/** @file -- MediaSanitizeUnitTest.c
+  Placeholder/framework for developing a Media Sanitize unit test package.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/UefiLib.h>
+#include <Library/DebugLib.h>
+#include <Library/UnitTestLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/UefiApplicationEntryPoint.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+#include <Uefi/UefiBaseType.h>
+#include <Library/UefiBootManagerLib.h>
+#include <Protocol/NvmExpressPassthru.h>
+
+#include "../NvmExpress.h"
+#include "../NvmExpressBlockIo.h"
+#include "../NvmExpressMediaSanitize.h"
+#include "../NvmExpressHci.h"
+#include "../../../Include/Protocol/MediaSanitize.h"
+
+// #pragma warning ( disable : 4100 )

```

Comment:
I had originally the NvmeRead() and NvmeWrite() functions as stubs (not doing anything inside). So I added the #pragma to suppress the unreferenced parameter warning. I've since fixed these two functions to address the parameters so I could remove the #pragma. I simply forgot to remove line #28. I will delete the line.